pkgdb2client/cli.py

```diff
@@ -985,8 +985,6 @@ def main():
     pkgdbclient.insecure = arg.insecure
 
     if arg.bzurl != pkgdb2client.BZ_URL:
-        if not arg.bzurl.endswith('xmlrpc.cgi'):
-            arg.bzurl = '%s/xmlrpc.cgi' % arg.bzurl
```

Comment:
Could you expand a little on why those changes? AFAIK the URL has to end with the xmlrpc.cgi endpoint no?